features/broadcast/distsql/handler/src/test/java/org/apache/shardingsphere/broadcast/distsql/handler/update/CreateBroadcastTableRuleStatementUpdaterTest.java

```diff
@@ -29,30 +31,47 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 class CreateBroadcastTableRuleStatementUpdaterTest {
     
     private final CreateBroadcastTableRuleStatementUpdater updater = new CreateBroadcastTableRuleStatementUpdater();
     
+    @Test
+    void assertCreatedRuleConfigurationWithNoStorageUnits() {
```

Comment:
This better name for this method is ` assertCheckSQLStatementXXX`

Reply:
ok